README.md

```diff
@@ -181,7 +184,7 @@ ReactDOM.render(
 
 
 ### <a name="getting-widget-instance"></a>Getting Widget Instance ###
-In some cases, a widget instance is required when you need to call a widget method. You can get it by assigning a callback function to the component's `ref` property. This function accepts the mounted DevExtreme Component as an argument whose `instance` property stores the widget instance.
+In some cases, you need a widget instance to call a widget method. You can get it by assigning a callback function to the component's `ref` property. This function accepts the mounted DevExtreme Component as an argument whose `instance` property stores the widget instance.
```

Comment:
Further down: "
 The components that displays content in an overlaying window (for example, [ScrollView]), allow to specify the content as component children:" Component**s** display**s**? Allow to specify? allow what to specify?

Reply:
as component children?